src/SmoFacade/Database.cs

```diff
@@ -14,6 +14,8 @@ public class Database
   {
     internal readonly Microsoft.SqlServer.Management.Smo.Database _database;
     private readonly Server _server;
+    private const int GB_TO_KB = 1048576;
```

Comment:
@alexirion10 Since the use for these is confined to `Database.cs` and since they're well-known values, I think a `private const` is more appropriate than `public static readonly`.

Reply:
Looks like there is a nuget package we could use instead, but I think it's owned by just one person (and not an organization) - so more risky

https://www.nuget.org/packages/ByteSize/
https://github.com/omar/ByteSize

Reply:
I don't see the need for an external nuget package when this can simply be a `const int` in here